packages/react-composites/src/composites/ChatComposite/FileDownloadCards.tsx

```diff
@@ -48,20 +52,34 @@ export const FileDownloadCards = (props: FileDownloadCards): JSX.Element => {
             fileName={truncatedFileName(file.name, truncateLength)}
             key={file.name}
             fileExtension={extension(file.name)}
-            actionIcon={<DownloadIconTrampoline />}
+            actionIcon={
+              showSpinner ? (
+                <Spinner size={SpinnerSize.medium} aria-live={'assertive'} />
+              ) : (
+                <ChatCompositeIcon iconName="Download" />
```

Comment:
Use DownloadIconTrampoline here (that was removed) as I assume stable build will fail without it

Reply:
@JamesBurnside I don't understand why stable build will fail without i?

Reply:
`Download` is a beta-only icon and icon names are now typeguarded

Reply:
Done